install.js

```diff
@@ -154,3 +158,23 @@ function logPolitely(toBeLogged) {
     console.log(toBeLogged);
 }
 
+function getCertificateAuthority(configCaFile) {
+  let ca;
```

Comment:
can you please add a comment pointing that this code is ported from phantomjs install script?